spec/support/feature/pikaday_support.rb

```diff
@@ -0,0 +1,11 @@
+module Feature
+  module PikadaySupport
+    def fill_in_pikaday_input(field_name, with:)
+      find_field(field_name).click
+      fill_in(field_name, with: with)
+      # This causes each Pikaday input to be correctly set
+      page.execute_script("$('body').focus()")
```

Comment:
I would suggest you you don't introduce a dependency on jQuery :) `document.getElementsByTagName('body')[0]`